http-gateway/web/public/web_configuration.json

```diff
@@ -1,15 +1,15 @@
 {
   "domain": "",
   "httpGatewayAddress": "",
-  "webOAuthClient": {
-    "clientID": "",
+  "webOauthClient": {
```

Comment:
@kuby do I need to change fields in https://github.com/plgd-dev/hub/blob/24bf149f5f1466b8d89f01de8828f231cbb97753/http-gateway/web/src/containers/things/provision-new-device.js ? I'm not familiar with that js syntax, but I think it may be json string translated into json object...?

Reply:
You have to change it in multiple files, I'll create a branch from this one and fix those occurrences, give me a sec.

Reply:
supes, thanks!

Reply:
https://github.com/plgd-dev/hub/pull/623

Reply:
merge to this branch please.

Reply:
OK, so after banging our heads and talking about this way too much, we've agreed on a new format for fields, yaml tags and json tags: camelCase, abbreviations and acronyms are capitalized, except when they are at the beginning of json/yaml tags. 

Example:
```
type PublicConfiguration struct {
	CAPool                   string       `yaml:"caPool" json:"caPool"
	HubID                    string       `yaml:"hubID" json:"hubID"`
```

I'll redo it accordingly, and we'll see if any changes in JS are necessary.